akka-docs/src/main/paradox/persistence-query-leveldb.md

```diff
@@ -1,38 +1,20 @@
 # Persistence Query for LevelDB
 
+## Dependency
+
+To use Persistence Query, you must add the following dependency in your project.
+
+@@dependency[sbt,Maven,Gradle] {
+  group=com.typesafe.akka
+  artifact=akka-persistence-query_$scala.binary_version$
+  version=$akka.version$
+}
+
```

Comment:
Mention that this will also make akka-persistence available, link to that module's docs